examples/start_heartbeat.rs

```diff
@@ -0,0 +1,45 @@
+use std::{
+    sync::{mpsc, Arc},
+    thread,
+    time::Duration,
+};
+
+use keygen_rs::{
+    config::{self, KeygenConfig},
+    errors::Error,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    config::set_config(KeygenConfig {
+        api_url: env::var("KEYGEN_API_URL").expect("KEYGEN_API_URL must be set"),
+        account: env::var("KEYGEN_ACCOUNT").expect("KEYGEN_ACCOUNT must be set"),
+        product: env::var("KEYGEN_PRODUCT").expect("KEYGEN_PRODUCT must be set"),
+        license_key: Some(env::var("KEYGEN_LICENSE_KEY").expect("KEYGEN_LICENSE_KEY must be set")),
+        public_key: Some(env::var("KEYGEN_PUBLIC_KEY").expect("KEYGEN_PUBLIC_KEY must be set")),
+        ..KeygenConfig::default()
+    });
+    let fingerprint = machine_uid::get().unwrap_or("".into());
+    if let Ok(license) = keygen_rs::validate(&[fingerprint.clone()], &[]).await {
+        let machine = license.machine(&fingerprint).await?;
+        let interval = Duration::from_secs(machine.heartbeat_duration.unwrap_or(570) as u64);
```

Comment:
The default `heartbeat_duration` is 60s.

see https://keygen.sh/docs/api/policies/#policies-object-attrs-heartbeatDuration

Reply:
The link says that 60s is the minimum heartbeat duration, I would be afraid to set something to the absolute minimum. Also the dropdown on app.keygen.sh when setting up a new policy does not even have an option to go under 2 minutes. 
Also here: https://keygen.sh/docs/api/machines/#machines-actions-ping it says that the default heartbeat ping window is 10 minutes. I have set it to 10 minutes - 30 seconds, similar to the go reference.